grails-app/assets/javascripts/asinha/customer-create.js

```diff
@@ -0,0 +1,87 @@
+// Labels
+var cpfCnpjLabelReference = $("#cpfCnpjLabel");
+
+// Inputs
+var addressInputReference = $("#address")[0];
+var addressNumberInputReference = $("#addressNumber")[0];
+var emailInputReference = $("#email")[0];
+var provinceInputReference = $("#province")[0];
+var cityInputReference = $("#city")[0];
+var stateInputReference = $("#state")[0];
+var cpfCnpjInputReference = $("#cpfCnpj")[0];
+var phoneInputReference = $("#phone")[0];
+var cepInputReference = $("#cep")[0];
+
+// Error message
+var errorMessageReference = $("#error")[0]
+
+// Inputmasks
+new Inputmask({mask: "999.999.999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+new Inputmask({mask: ["(99) 9999-9999", "(99) 99999-9999"], showMaskOnHover: false, keepStatic: true}).mask(phoneInputReference);
+new Inputmask({mask: "99999-999", showMaskOnHover: false, keepStatic: true}).mask(cepInputReference);
+
+// Radio Buttons
+var cpfRadioReference = $("#cpfRadio");
+var cnpjRadioReference = $("#cnpjRadio");
+
+cpfRadioReference.click( function() {
+    cpfCnpjLabelReference.html("CPF");
+    new Inputmask({mask: "999.999.999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+})
+
+cnpjRadioReference.click( function() {
+    cpfCnpjLabelReference.html("CNPJ");
+    new Inputmask({mask: "99.999.999/9999-99", showMaskOnHover: false, keepStatic: true}).mask(cpfCnpjInputReference);
+})
+
+// Cep Handler
+var preeencherFormulario = (endereco) => {
+    addressInputReference.value = endereco.logradouro;
+    provinceInputReference.value = endereco.bairro;
+    cityInputReference.value = endereco.localidade;
+    stateInputReference.value = endereco.uf;
+    addressNumberInputReference.focus();
+}
+
+var cepCleaner = function(cep) {
+    var cleanCep = cep.replace(/\D/g, '')
+    return cleanCep    
+}
+
+var limpaFormulario = () => {
+    addressInputReference.value = "";
+    provinceInputReference.value = "";
+    cityInputReference.value = "";
+    stateInputReference.value = "";
+}
+    
+var pesquisarCep = async() => {
+    const cep = cepInputReference.value;
+    if (cepCleaner(cep).length == 8) {
+        const endpoint = `http://viacep.com.br/ws/${cep}/json/`;
+        let params = { method: "get", body: null };
+        fetch(endpoint, params)
+        .then(result => result.json())
+        .then(function (dados) { 
+            let error = ("erro" in dados);
+            if (!error) {
+                document.getElementById("error").innerHTML = "";
```

Comment:
no início do arquivo você captura um elemento e guarda na variável `errorMessageReference` e agora está buscando o elemento de novo... a ideia não seria usar a variável?